charts/redpanda/templates/statefulset.yaml

```diff
@@ -115,7 +115,7 @@ spec:
             - name: datadir
               mountPath: /var/lib/redpanda/data
   {{- if get .Values.statefulset.initContainers.fsValidator "resources" }}
-          resources: {{- toYaml .Values.statefulset.fsValidator.tuning.resources | nindent 12 }}
+          resources: {{- toYaml .Values.statefulset.fsValidator.resources | nindent 12 }}
```

Comment:
Strange this was the only incorrect one 🤔

Reply:
fsValidator was added as the last init container in the whole Statefulset history https://github.com/redpanda-data/helm-charts/pull/869